lib/actions/preload.coffee

```diff
@@ -143,9 +171,9 @@ module.exports =
 			alias: 's'
 		}
 		{
-			signature: 'dont-check-device-type'
+			signature: 'dont-check-arch'
```

Comment:
Is this a breaking change? I suspect so, since any users using the previous argument will suddenly find that checks _are_ happening (though the checks themselves have changed).

Are there any possible cases where they could be doing something that would work, that will now start failing?

If not, this is ok I think. If so though, we'll need to retarget this to the `v8-meta-branch` branch, which has a couple of other breaking changes @CameronDiver is planning to release, and merge this PR manually, and then merge that branch later to release these breaking changes together.

Reply:
Ok. We should retarget, merge manually (with no-checks), and release along with Cameron's changes then.

Reply:
Just-an-idea: If we prefer having this as a minor release, then we could just add `dont-check-arch` and since it sounds like it would have the same effect as `dont-check-device-type`, since in both cases, we don't use the device type at all. In that case I guess we could also add a deprecation warning in the docs for `dont-check-device-type`.